resources/dictionaries/violations.csv

```diff
@@ -1234,6 +1234,77 @@
 "NJ","39:8-1 VEHICLE INSPECTION","VEHICLE INSPECTION","Equipment"
 "NJ","39:8-4 REPAIRS OF VEHICLES","REPAIRS OF VEHICLES","Other"
 "NJ","39:8-6 DISPLAY OF INSPECTION","DISPLAY OF INSPECTION","Paperwork"
+"NY","VTL1180     0D    0I0","SPEED IN ZONE","Speeding"
+"NY","VTL1180     0B    0I0","SPEED OVER 55 ZONE","Speeding"
+"NY","VTL0306     B     0I0","UNINSPECTED MOTOR VEHICLE","Paperwork"
+"NY","VTL1229-C   03    0I0","NO SEAT BELT","Seat belt"
+"NY","VTL0509     01    0I0","UNLICENSED OPERATOR","License"
+"NY","VTL0375     2A1   0I0","NO/INADEQUATE LIGHTS","Lights"
+"NY","VTL1110     0A    0I0","DISOBEYED TRAFFIC CONTROL DEVICE","Stop sign/light"
+"NY","VTL1225-C   02A   0I0","OP MV- MOBILE PHONE","Cell phone"
+"NY","VTL0402     01    0I0","NO DISTINCTIVE PLATE/INSECURE/DIRTY","Registration/plates"
+"NY","VTL0375     40    0I0","INADEQUATE OR NO STOP LAMPS","Equipment"
+"NY","VTL0401     01A   0I0","UNREGISTERED MOTOR VEHICLE","Registration/plates"
+"NY","VTL0375     12A2  0I0","SIDEWINGS/SIDEWINDOWS/NON/TRASNPRNT","Equipment"
+"NY","VTL0375     2A3   0I0","NO/INSUFFICIENT TAIL LAMPS","Lights"
+"NY","VTL0511     01A   UM3","AGGRAVATED UNLIC OP 3RD (MISD)","License"
+"NY","VTL1128     0A    0I0","MOVED FROM LANE UNSAFELY","Safe movement"
+"NY","VTL1225-D   01    0I0","OPER MV WHILE USING PORTABLE ELEC DEV","Cell phone"
+"NY","VTL0375     2A4   0I0","NO/INADEQUATE PLATE LAMPS","Lights"
+"NY","VTL0375     31    0I0","INADEQUATE/NO MUFFLER","Equipment"
+"NY","VTL1192     03    UM0","DRIVING WHILE INTOXICATED (MISD)","DUI"
+"NY","VTL0509     08    0I0","FLD/NOTIFY DMV CHANGE OF ADDRESS","Paperwork"
+"NY","VTL1163     0D    0I0","IMPROPER/NO SIGNAL","Safe movement"
+"NY","VTL1129     0A    0I0","FOLLOWING TOO CLOSELY","Safe movement"
+"NY","VTL1144-A   A     0I0","FLD DUE CARE FOR EMERG VEH STOPPED OR STANDING","Safe movement"
+"NY","TL 140      2CII  0I0","NO FIRE EXTINGUISHER","Truck"
+"NY","VTL1180     0F    0I0","SPEEDING IN POSTED WORK ZONE","Speeding"
+"NY","TL 140      2DII  0I0","NO FIRE EXTINGUISHER","Truck"
+"NY","VTL0375     42    0I0","UNLAWFUL SPEEDOMETER","Equipment"
+"NY","VTL1192     02    UM0","DRIV'G W/.08 OF 1% OR  MORE OF ALC (MISD)","DUI"
+"NY","VTL0375     30    0I0","DRIVERS VIEW OBSTRUCTED","Equipment"
```

Comment:
Could also be considered "Safe movement" ? A bit ambiguous...

Reply:
sure, I'll change it.

Reply:
actually, we have been inconsistently labeling it throughout. Most states have it labeled as Equipment, a few as Safe Movement. I've changed them all to the former as that seems to have been the consensus so far. Also, sometimes it is a hybrid field like "Tinted or obstructed windshield". I can change it back if you think it's more accurately described as Safe Movement.